config/routes.rb

```diff
@@ -28,6 +28,14 @@
     scope module: :schools do
       resources :contacts
       resources :alerts
+
+      resources :meters do
+        member do
```

Comment:
I'd considered adding these as one or two separate controllers instead of members. Any preference?

Reply:
as an overall preference, I like my controllers to have the usual restie kind of actions - but I'm happy to keep them here for now. I would be interested to see what those two separate controllers would be like though - would they just both have update actions?

Reply:
A few options, off the top of my head:
`Schools::ActivationController#create` and `Schools::DectivationController#create`
or
`Schools::ActiveStatusController#update`
or
`Schools::ActivationController#create` and `Schools::ActivationController#destroy`

My preference would be for at least two separate actions, so not the middle option. All depends on which ones are more RESTy.

Reply:
Yes, I think I'm with you on 1 or 3 - I think 3 might be my preference.. although is it anything to do with Schools ? Should it be Meters::ActivationController ? ;)